examples/flink-python-example/README.md

```diff
@@ -0,0 +1,83 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Flink Kubernetes Operator Python Example
+
+## Overview
+
+This is an end-to-end example of running Flink Python jobs using the Flink Kubernetes Operator.
+
+
+*What's in this example?*
+
+ 1. Python script of a simple streaming job
```

Comment:
Here we are just showing a workable example and IMO there is no key difference between running a pyflink batch/streaming job.

But you do raise a good point. I have tried batch examples in pyflink as well. And I notice that in [FLINK-27468](https://issues.apache.org/jira/browse/FLINK-27468), we add `effectiveConfig.set(SHUTDOWN_ON_APPLICATION_FINISH, false);` and as a result, when the batch example finishes, the job manager pod will not exit. Such behavior is subtly different the directly running `flink run-application ....` which release JM pods in the end.
Our user story of how to support batch jobs may need more discussion cc@gyfora @wangyang0918  but it may be not in the scope of this PR.

Reply:
@bgeng777, if the user uses the example to create table environment with `StreamTableEnvironment.create(EnvironmentSettings.inBatchMode())`, what's the behavior will happen?

Reply:
@SteNicholas The job will finish normally but the JM will keep running after the batch job finishes.